security/pkg/stsservice/sts.go

```diff
@@ -94,9 +96,10 @@ type TokenManager interface {
 
 // TokenInfo stores token information maintained at TokenManager.
 type TokenInfo struct {
-	TokenType  string `json:"token_type"`
-	IssueTime  string `json:"issue_time"`
-	ExpireTime string `json:"expire_time"`
+	TokenType  string    `json:"token_type"`
+	IssueTime  time.Time `json:"issue_time"`
+	ExpireTime time.Time `json:"expire_time"`
+	Token      string    `json:"token"`
```

Comment:
Is tokeinfo printed out only at debug time?

Reply:
This is not used for debug, when we turn on debug, this is not printed. Only used for caching.